### PR TITLE
Remove unused legacy JS from the legacy directory.

### PR DESCRIPTION
**Pull request by dennisoelkers**
Removing bits from the legacy folder that I could identify as being unused/replaced.

### CODE REVIEW

File: graylog2-web-interface/webpack/vendor-module-ids.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "nextFreeModuleId": 586,
```
Comment (joschi) [inline graylog2-web-interface/webpack/vendor-module-ids.json]: Is this supposed to be committed?

File: graylog2-web-interface/webpack.vendor.js [hunk 2]
```diff
@@ -13,8 +13,6 @@ console.log('Building vendor bundle.');
 
 const webpackConfig = {
   entry: {
-    //app: APP_PATH,
-    //config: 'config.js',
```
Comment (joschi) [inline graylog2-web-interface/webpack.vendor.js]: Please remove those attributes instead of just commenting them out.

**Comment by joschi** [comment]
@dennisoelkers Could you please rebase this PR on the current `master` branch?

**Comment by dennisoelkers** [comment]
:ballot_box_with_check:

**Comment by joschi** [comment]
Thanks!

**Pull request merged**